### PR TITLE
Small improvements to systemd-creds output and man page

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: src/shared/creds-util.c [hunk 1]
```diff
@@ -236,61 +237,78 @@ int get_credential_host_secret(CredentialSecretFlags flags, void **ret, size_t *
...
                 }
 
                 if (fstat(fd, &st) < 0)
-                        return -errno;
+                        return log_debug_errno(errno, "Failed to stat %s/%s: %m", dirname, filename);
 
                 r = stat_verify_regular(&st);
                 if (r < 0)
-                        return r;
+                        return log_debug_errno(errno, "Failed to stat %s/%s: %m", dirname, filename);
```
Comment (yuwata) [inline src/shared/creds-util.c]: `errno` -> `r`, and maybe the log message should be adjust to mention that `stat_verify_regular()` is failed?

**Review by yuwata — CHANGES_REQUESTED** [review]

File: src/shared/creds-util.c [hunk 2]
```diff
@@ -183,49 +183,50 @@ static int make_credential_host_secret(
...
+        dfd = open_mkdir_at(AT_FDCWD, dirname, O_CLOEXEC, 0755);
         if (dfd < 0)
-                return dfd;
+                return log_debug_errno(dfd, "Failed to create or open directory '%s': %m", dirname);
 
         if (FLAGS_SET(flags, CREDENTIAL_SECRET_FAIL_ON_TEMPORARY_FS)) {
                 r = fd_is_temporary_fs(dfd);
                 if (r < 0)
-                        return r;
+                        return log_debug_errno(dfd, "Failed to check directory '%s': %m", dirname);
```
Comment (bluca) [inline src/shared/creds-util.c]: return `r`?

Comment (keszybz) [reply src/shared/creds-util.c]: Oops.

File: src/shared/creds-util.c (hunk 1 above)
Comment (keszybz) [reply src/shared/creds-util.c]: Oops.

**Review by bluca — APPROVED** [review]

**Pull request merged**